piqueserver/extensions.py

```diff
@@ -0,0 +1,101 @@
+
+import os
+import importlib
+import imp
+
+def check_scripts(scripts):
+    '''
+    Ensures that there are no duplicate scripts in scripts
+    :param scripts: list of scripts
+    :return: True if there is no duplicate scripts, False otherwise
+    '''
+    seen = set()
+    dups = []
+    for script in scripts:
+        if script in seen:
+            dups.append(script)
+        else:
+            seen.add(script)
+    if dups:
+        #log.warn("Scripts included multiple times: {}".format(dups)) TODO: Pass logger object to this function
```

Comment:
You should just create a new logger instance here. This is important because it will show the file that the message comes from. You can check existing files to see how to do this.

Reply:
Fixed!